app/jobs/destroy_collections_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# app/jobs/destroy_collections_job.rb
+class DestroyCollectionsJob < ApplicationJob
+  def perform
+    # Find specific collection
+    Collection.find_each do |collection|
+      next if collection.title.first == 'Journal of Adventist Education'
+
+      # Destroy the collections
+      collection.destroy!
+      Rails.logger.debug "#{collection.title.first} was destroyed!"
```

Comment:
We may want to include a rescue statement here as well, so that the job will continue running even if there's an error. 

some ideas to reference: